Pages/MainPage.cs

```diff
@@ -0,0 +1,128 @@
+﻿using OpenQA.Selenium;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Pages
+{
+    public class MainPage
+    {
+        private static IWebDriver _driver;
+        public MainPage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+
+        public List<IWebElement> GetDivElements()
+        {
+            List<IWebElement> DivElements = _driver.FindElements(By.TagName("div")).ToList();
+            return DivElements;
+        }
+
+        public List<IWebElement> GetDivH1Elements()
+        {
+            List<IWebElement> DivH1Elements = _driver.FindElements(By.XPath(".//div//h1")).ToList();
+            return DivH1Elements;
+        }
+
+        public List<IWebElement> GetNewsElements()
+        {
+            List<IWebElement> NewsElements = _driver.FindElements(By.XPath(".//div[@class='readmore_item']")).ToList();
+            return NewsElements;
+        }
+
+        public IWebElement GetSpanElement()
+        {
+            IWebElement CityElement = _driver.FindElement(By.XPath(".//div/a/span[@class='cities_name'][text()='Чернигов']"));         
+            return CityElement;
+        }
+
+        public List<string> GetNewsElementsTitles()
+        {
+            List<IWebElement> NewsElements = _driver.FindElements(By.XPath(".//div[@class='readmore_item']/div/a/div[starts-with(@style,'back')]")).ToList();
+            List<string> elementTitles= new List<string>();
+            foreach(IWebElement element in NewsElements)
+            {
+                elementTitles.Add(element.Text);
+            }
+            return elementTitles;
+        }
+
+        public IWebElement GetElementsWithKyivTitle()
+        {
+            IWebElement CityElement = _driver.FindElement(By.XPath(".//span[text()='Киев']"));
```

Comment:
it is possible that the language on the page will be different. could you please handle that?